### PR TITLE
Fix greetings

### PR DESCRIPTION
**Pull request by unseeyou**
I realise that if you press the button a second time then it is still deferred from the first time or something since InteractionResponded is getting raised

### CODE REVIEW

**Comment by unseeyou** [comment]
this also only allows the person who send the command to click the button

**Comment by unseeyou** [comment]
uhh

**Comment by unseeyou** [comment]
crossed off some TODOs

File: bot/bot.py [hunk 1]
```diff
@@ -44,3 +48,16 @@ async def connect_to_database(self) -> None:
...
+        self,
+        interaction: ApplicationCommandInteraction,
+        exception: errors.CommandError,
+    ) -> None:
+        self.logger.exception(exception)
+        with contextlib.suppress(disnake.errors.InteractionResponded):
+            await interaction.response.defer()
+        if isinstance(exception, bot.errors.DatabaseNotConnectedError):
+            await interaction.followup.send("Database is not connected", ephemeral=True)
+            return
```
Comment (interrrp) [inline bot/bot.py:62]: We should move the error handling into an extension/cog, but that's for later

**Review by interrrp — APPROVED** [review]

**Pull request merged**